### DISCUSSION TITLE
Resume CDT::Triangulation

### DISCUSSION THREAD

**Discussion by aeioaeu** [artem-ogre/CDT#184] [General]
Hello. Is it possible to resume CDT::Triangulation from a previous meshing?
After finalizing CDT::Triangulation and extracting the vertex list and triangle list, I would like to add a new vertex list and perform CDT::Triangulation.
Even if this is not possible at the moment, I would like to hear any good ideas.
Thank you for making a great library.
Cheers

**Comment by artem-ogre** [comment]
Hello, @aeioaeu 👋 
`CDT::Triangulation` class is designed to be copyable. As a work around you can finalize a copy and add new vertices to the original. Can this be a suitable approach for you?

**Comment by aeioaeu** [comment]
Thanks for the reply.
It's not sufficient for my use case because I want to serialize a vertex list and a triangle list.

**Comment by artem-ogre** [comment]
I don’t think I follow. Can you give some more details, an example?

**Comment by aeioaeu** [comment]
For example, I want to import a mesh created in Blender into ASSIMP and add breaklines to that mesh.
